packages/office-ui-fabric-react/src/components/Slider/Slider.base.tsx

```diff
@@ -109,7 +114,30 @@ export class SliderBase extends BaseComponent<ISliderProps, ISliderState> implem
               {originFromZero && (
                 <span className={css(classNames.zeroTick)} style={this._getStyleUsingOffsetPercent(vertical, zeroOffsetPercent)} />
               )}
-              <span ref={this._thumb} className={classNames.thumb} style={this._getStyleUsingOffsetPercent(vertical, thumbOffsetPercent)} />
+              {typeof marks === 'boolean' && marks && this._addTickmarks(css(classNames.regularTick))}
+              {thumblabel ? (
+                <TooltipHost
+                  content={'' + value}
+                  id={this._hostId}
+                  calloutProps={{ gapSpace: 5, beakWidth: 8, target: `#${this._buttonId}`, doNotLayer: true }}
+                >
+                  <span
+                    ref={this._thumb}
+                    className={classNames.thumb}
+                    style={this._getStyleUsingOffsetPercent(vertical, thumbOffsetPercent)}
+                    id={this._buttonId}
+                    aria-labelledby={this._hostId}
+                  />
+                </TooltipHost>
+              ) : (
+                <span
```

Comment:
the two spans look identical despite being fairly substantial. can you avoid the code duplication?